new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/DeployUtil.java

```diff
@@ -0,0 +1,265 @@
+          .and().with().pollInterval(10, SECONDS)
+          .atMost(5, MINUTES).await();
+
+  /**
+   * Deploy application.
+   *
+   * @param host name of the admin server host
+   * @param port default channel node port of admin server
+   * @param userName admin server user name
+   * @param password admin server password
+   * @param targets comma separated list of targets to deploy applications
+   * @param archivePath local path of the application archive
+   * @param namespace name of the namespace in which WebLogic server pods running
+   */
+  public static void deployApplication(String host, String port, String userName,
+      String password, String targets, Path archivePath, String namespace) {
+
+    setImage(namespace);
+
+    // create a temporary WebLogic domain property file
+    File domainPropertiesFile = assertDoesNotThrow(() -> File.createTempFile("domain", "properties"),
+        "Creating domain properties file failed");
+    Properties p = new Properties();
+    p.setProperty("node_archive_path", MOUNT_POINT + archivePath.getFileName());
+    p.setProperty("admin_host", host);
+    p.setProperty("admin_port", port);
+    p.setProperty("admin_username", userName);
+    p.setProperty("admin_password", password);
+    p.setProperty("targets", targets);
+    assertDoesNotThrow(() -> p.store(new FileOutputStream(domainPropertiesFile), "wlst properties file"),
+        "Failed to write the domain properties to file");
+
+    // WLST py script for deploying application
+    Path deployScript = Paths.get(RESOURCE_DIR, "python-scripts", DEPLOY_SCRIPT);
+
+    logger.info("Creating a config map to hold deployment files");
+    String deployScriptConfigMapName = "create-deploy-scripts-cm";
+
+    Map<String, String> data = new HashMap<>();
+    Map<String, byte[]> binaryData = new HashMap<>();
+    assertDoesNotThrow(() -> {
+      data.put(DEPLOY_SCRIPT, Files.readString(deployScript));
+      data.put(DOMAIN_PROPERTIES, Files.readString(domainPropertiesFile.toPath()));
+      binaryData.put(archivePath.getFileName().toString(),
+          Base64.getMimeEncoder().encode(Files.readAllBytes(archivePath)));
+    });
+
+    V1ObjectMeta meta = new V1ObjectMeta()
+        .name(deployScriptConfigMapName)
+        .namespace(namespace);
+    V1ConfigMap configMap = new V1ConfigMap()
+        .data(data)
+        .binaryData(binaryData)
+        .metadata(meta);
+
+    assertDoesNotThrow(() -> createConfigMap(configMap),
+        String.format("Failed to create configmap %s with files", configMap));
+
+    // deploy application with deploy scripts and domain properties on persistent volume
+    deploy(namespace, deployScriptConfigMapName);
+  }
+
+  /**
+   * Deploy application by creating a job.
+   *
+   * @param namespace namespace in which to create job
+   * @param deployScriptConfigMapName configmap containing deployment scripts
+   */
+  private static void deploy(String namespace, String deployScriptConfigMapName) {
```

Comment:
Can u change modify the name of function to be specific instead of generic deploy()  e.g. deployUsingWlst()

Reply:
changed the method name